packages/components-react/src/heading/Heading.tsx

```diff
@@ -0,0 +1,47 @@
+import { Heading as UtrechtHeading } from '@utrecht/component-library-react/dist/css-module';
+import { HTMLAttributes, PropsWithChildren, ReactNode } from 'react';
+
+type HeadingLevels = 1 | 2 | 3 | 4 | 5 | 6;
+
+export interface LuxHeadingProps extends HTMLAttributes<HTMLHeadingElement> {
+  appearance?: HeadingLevels;
+  level: HeadingLevels;
+}
+
+export const LuxHeading = ({
+  children,
+  level = 6,
+  appearance = level,
+  ...props
+}: PropsWithChildren<LuxHeadingProps>): ReactNode => {
+  const _level = level > 6 || level < 1 ? 6 : level;
```

Comment:
```suggestion
  const _level = Math.min(level, 6);
```

The goal here is to have it be level 6, at maximum. This is a more readable and maintainable version

Reply:
If you agree, also apply it to the next line

Reply:
Also, maybe we don't need it as we have already told TypeScript that it can only be 1-6

Reply:
Yeah, that's true. TypeScript should prevent this. I added this to prevent issues at runtime (in the case that someone bypasses / ignores TypeScript somehow) .